include/battle_anim.h

```diff
@@ -230,6 +231,8 @@ void AnimMudSportDirt(struct Sprite *sprite);
 void AnimDirtScatter(struct Sprite *sprite);
 void AnimMudSportDirtRising(struct Sprite *sprite);
 void AnimDirtPlumeParticle(struct Sprite *);
+void AnimBoneHitProjectile(struct Sprite *);
```

Comment:
```suggestion
void AnimBoneHitProjectile(struct Sprite *sprite);
```
All new things should at least have the input declared.